src/components/DropZone/DropZone.tsx

```diff
@@ -363,18 +382,29 @@ export class DropZone extends React.Component<CombinedProps, State> {
       return;
     }
 
-    let size = 'extraLarge';
-    const width = this.node.getBoundingClientRect().width;
-
-    if (width < 100) {
-      size = 'small';
-    } else if (width < 160) {
-      size = 'medium';
-    } else if (width < 300) {
-      size = 'large';
+    let height;
+    let width;
+    const wrapperHeight = this.node.getBoundingClientRect().height;
+    if (wrapperHeight < Size.Small) {
```

Comment:
Since this if statement and the below if statement are the same we could turn it into a function to reduce code, but I don't have strong feelings about this since. So whatever you feel is better is 👌

Reply:
I like that idea. I tried out a function.  Let me know what you think.